src/anonfile/anonfile.py

```diff
@@ -319,7 +319,7 @@ def preview(self, url: str, path: Union[str, Path]=Path.cwd()) -> ParseResponse:
         from anonfile import AnonFile
 
         anon = AnonFile()
-        preview = anon.preview('https://anonfiles.com/b7NaVd0cu3/topsecret_mkv')
+        preview = anon.preview('https://anonfiles.com/P0mev3tfz7/topsecret_mp4')
```

Comment:
It's nice of you that you also updated the comment here ;)

though it wouldn't hurt anyone if the URL didn't work anymore, it's just an example